src/components/FlowRunTimeline.vue

```diff
@@ -215,11 +215,12 @@
   })
 
   const api = useWorkspaceApi()
+  const interval = isRunningStateType(props.flowRun.stateType) ? { interval: 5000 } : undefined
```

Comment:
This will maintain a 5 second interval for any runs that transition from non-terminal states during the lifetime of the component, is that ok?

Reply:
@znicholasbrown take a look at the state type watcher. My thought is if the flowRun enters a terminal state
- Refresh immediately (get the final data)
- Unsubscribe (data cannot change anymore)
That will stop the interval

Reply:
ty!!